geode-core/build.gradle

```diff
@@ -177,6 +177,7 @@ dependencies {
   compileOnly files("${System.getProperty('java.home')}/../lib/tools.jar")
 
   //Find bugs is used in multiple places in the code to suppress findbugs warnings
+  compileOnly("io.swagger:swagger-annotations")
```

Comment:
Why is this needed in geode-core?  AFAICT, the swagger annotations are not used in geode-core.  There's also a runtime dependency declaration.  Which one is correct?  Note that the runtime declaration is also marked as optional.

Reply:
just testing this out. Will revert this.